application/common/models/Mfa.php

```diff
@@ -453,19 +453,25 @@ public function getReadableType()
      */
     public static function removeOldUnverifiedRecords()
     {
-        /*
-         * Replace '+' with '-' so all env parameters can be defined consistently as '+n unit'
-         */
-        $mfaLifetime = str_replace('+', '-', \Yii::$app->params['mfaLifetime']);
+        self::deleteOldRecords(\Yii::$app->params['mfaLifetime'], ['verified' => 0]);
+    }
+
+    /**
+     * @param string $age PHP relative time denoting how old a record must be to qualify for removal
+     * @param array $criteria array suitable for passing to a query, like ['field' => value]
+     */
+    protected static function deleteOldRecords(string $age, array $criteria): void
+    {
+        $age = '-' . ltrim($age, '+');
```

Comment:
I think you'll want to keep this `+` to `-` replacement wherever you retrieve the `mfaLifetime` parameter (e.g. up in the `removeOldUnverifiedRecords()` method), so that this `deletedOldRecords()` method's `$age` argument will always be specified as `-something`.

I suspect that it'll also work out better to continue using the `str_replace('+', '-', ...)` approach rather than the `'-' . ltrim(..., '+')` approach, as the former can handle values specified with either a + or a -, but the latter can only handle values specified with a + (if there is a -, it would be doubled).